osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -76,7 +76,7 @@ protected ScoreProcessor()
             Combo.ValueChanged += delegate { HighestCombo.Value = Math.Max(HighestCombo.Value, Combo.Value); };
         }
 
-        private ScoreRank rankFrom(double acc)
+        public static ScoreRank RankFrom(double acc)

```

Comment:
move this inside of score. we don't use static public methods in non-static classes